src/views/Projects/index.jsx

```diff
@@ -0,0 +1,66 @@
+import { Component, Fragment } from 'react';
+import { withStyles } from 'material-ui/styles';
+import Grid from 'material-ui/Grid';
+import Typography from 'material-ui/Typography';
+import projects from '../../data/loader';
+import ProjectCard from '../../components/ProjectCard';
+
+@withStyles(
+  theme => ({
+    content: {
+      flexGrow: 1,
+      minWidth: 0,
+    },
+    container: {
+      paddingRight: '15px',
+      paddingLeft: '15px',
+      marginRight: 'auto',
+      marginLeft: 'auto',
+    },
+    header: {
+      paddingTop: theme.spacing.unit,
+      paddingBottom: 2 * theme.spacing.unit,
+    },
+  }),
+  { withTheme: true }
+)
+export default class Projects extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      projects,
+    };
+  }
```

Comment:
A constructor is not needed here. You can simply initialize the state as a class property:

```jsx
export default class Projects extends Component {
  state = {
    projects,
  };

  // ...
}
```